CHANGELOG.md

```diff
@@ -131,6 +131,7 @@
 - PR #3871 Fix `split_out` error
 - PR #3886 Fix string column materialization from column view
 - PR #3893 Parquet reader: fix segfault reading empty parquet file
+- PR #3931 Groupby agg structure regression affecting dask
```

Comment:
This describes a bug, not a fix.  changelog (and PR title) should always describe the fix.
```suggestion
- PR #3931 Fix groupby agg structure regression affecting dask
```